src/containers/Tokens/TokensTable/tests/hooks.test.js

```diff
@@ -20,14 +19,6 @@ const mountTable = (isError = false, allTokens = []) => {
 };
 
 describe('Testing hooks', () => {
-  beforeEach(() => {
```

Comment:
Was this no longer being used?

Reply:
Nope, legacy stuff that I noticed while trying to fix tests for one of the other components